src/libraries/Common/tests/System/Security/Cryptography/AlgorithmImplementations/ECDsa/ECDsaTests.netcoreapp.cs

```diff
@@ -188,7 +188,6 @@ public void KeySizeProp()
         }
 
         [Theory, MemberData(nameof(TestNewCurves))]
-        [ActiveIssue("https://github.com/dotnet/runtime/issues/51332", TestPlatforms.iOS | TestPlatforms.tvOS | TestPlatforms.MacCatalyst)]
```

Comment:
This seems unrelated from AES-GCM and ChaCha20Poly1305. Does this test pass on those platforms?

Reply:
Yes it passes.